runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/ProcessInstance/ProcessInstance.reducers.ts

```diff
@@ -6,14 +6,35 @@ import { initialState } from './ProcessInstance.slice';
 import { ProcessInstanceState } from './ProcessInstance.state';
 import { updateProcessInstanceProps } from './ProcessInstance.utils';
 
+interface MapOrchestrator {
+    processId: number,
+    orchestratorProcessInstanceId: string
+}
+
 export const updateOrchestratorProcessInstanceId = (state: ProcessInstanceState, action: PayloadAction<string>) => {
     state.active.orchestratorProcessInstanceId = action.payload;
 };
 
+export const updateOrchestratorProcessInstanceIdMap = (state: ProcessInstanceState, action: PayloadAction<MapOrchestrator>) => {
+    state.allActiveMap[action.payload.processId] = {
+        ...(state.allActiveMap[action.payload.processId]),
+        orchestratorProcessInstanceId: action.payload.orchestratorProcessInstanceId
+    };
+
+};
```

Comment:
imo when we add new `orchestratorProcessInstanceId` that means that new process has started its execution. Thus, we should not keep any other informations about the outdated processInstance